README.md

```diff
@@ -151,12 +151,10 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
 
 ### Using `npm`:
 
-1. If you don't already have npm on your machine, install it. On OSX with [Homebrew](https://brew.sh/) that looks like:
+1. If you don't already have npm on your machine, install it using [npm's recommended method](https://www.npmjs.com/get-npm).
+
+    If you have already installed npm with `brew` or `apt`, it is possible you will run into an `EACCES` error (related to how many system packagers install node) while installing wrangler. You can either uninstall that version of node and use the npm recommended install method, or use `cargo` instead.
```

Comment:
alternatively we can point people to the releases tab. they can manually download the binary for their machine and place it where they'd like

Reply:
I am concerned about recommending installing a raw binary that can't update itself

Reply:
i hear you. i suggest it because its an option for people who are really stuck. in my workshop, there was one person who had to do it this way. since it exists i think it . is worth mentioning, as well as the building from source. 

we would call this a "manual install" and this is a common pattern for open source software.

Reply:
```suggestion
    If you have already installed npm with a package manager, it is possible you will run into an `EACCES` error (related to how many system packagers install npm) while installing wrangler. You can either uninstall that version of npm and use the npm recommended install method (a version manager), use `cargo` instead, or download a binary from [our releases page ](https://github.com/cloudflare/wrangler/releases) and put it on your PATH.
```

Reply:
@ashleymichal Look good to you?

Reply:
it's gotten way dense. it looks like it's worth calling out "manual install" in its own section after cargo. then all you have to say here is "you can either uninstall and reinstall as instructed above, or use an alternate wrangler installation method", call out nvm above (now i see why that is valuable, my bad for the earlier comment), and